Amplify/Categories/API/Request/RESTRequest.swift

```diff
@@ -32,9 +32,13 @@ public class RESTRequest {
                 headers: [String: String]? = nil,
                 queryParameters: [String: String]? = nil,
                 body: Data? = nil) {
+        if headers?["Cache-Control"] == nil {
+            self.headers = ["Cache-Control": "no-store"]
```

Comment:
🤔  wouldn't this be a problem if we pass a `header` parameter that does not have  a `"Cache-Control` value, but it does have others values?

For example, if we just provide `["content-type": "application/json"]` to the init, we'd end up with only `["Cache-Control": "no-store"]`, right?

Reply:
Fixed